aula-pratica-02/ex2.c

```diff
@@ -0,0 +1,28 @@
+#include<stdio.h>
+#include<stdlib.h>
+int primo(int n);
+
+int primo(int n)
+{
+    for (int i = 2; i < n; i ++) {
+        if (n % i == 0) {
+            return 0;
+        }
+    }
+    return 1;
+}
+
+int main()
+{
+    int n, soma = 0, cont = 0;
+    scanf("%d", &n);
+    for(int i = 2; cont < n; i++){
+        if(primo(i) == 1){

```

Comment:
here as the 1 and 0 is a truthy and falsy values (respectively) you can omit the `== 1`
```suggestion
        if(primo(i)){
```

also if you change the function signature (return type and name) like I suggested, you kinda have to take this out, otherwise you'd endup with something like `if (isPrime(i) == true)` which is just silly/reduntant